### DISCUSSION TITLE
Default Graph (gremlin) not available through studio

### DISCUSSION THREAD

**Discussion by Praveenstein** [ArcadeData/arcadedb#659] [Q&A]
I've stated arcadedb with gremlin server enabled using docker, and i can see that the default graph (database instance) is configured both in arcade db and gremlin server, but when i try accessing from the studio it is not available, only the "Imported" graph (database instance)  is available, please see the attached screen shots
![Screenshot 2022-12-08 112645](https://user-images.githubusercontent.com/45847217/206369458-bda9bb7d-edb5-4b11-ad0c-f3109a349dce.png)
![Screenshot 2022-12-08 112812](https://user-images.githubusercontent.com/45847217/206369471-557839f8-063e-4b7d-a898-9ab9f55e5c1e.png)

**Comment by lvca** [comment]
If you look into the directory `$ARCADEDB_HOME/databases` which databases do you see? Each database has its own directory.

**Comment by Praveenstein** [comment]
Here is the screenshot from docker cli, it shows two directories (one for Imported and one for graph) but in studio as i mentioned only Imported is available
![Screenshot 2022-12-08 114442](https://user-images.githubusercontent.com/45847217/206371802-0493a03e-3171-45ed-8571-b02c2c26f247.png)

**Reply by Praveenstein** [reply]
and i used this command to start the docker `` docker run --rm  -p 2480:2480 -p 2424:2424 -p 6379:6379 -p 5432:5432 -p 8182:8182 --env JAVA_OPTS="-Darcadedb.server.rootPassword=playwithdata -Darcadedb.server.defaultDatabases=Imported[root]{import:https://github.com/ArcadeData/arcadedb-datasets/raw/main/orientdb/OpenBeer.gz} -Darcadedb.server.plugins=Redis:com.arcadedb.redis.RedisProtocolPlugin,MongoDB:com.arcadedb.mongo.MongoDBProtocolPlugin,Postgres:com.arcadedb.postgres.PostgresProtocolPlugin,GremlinServer:com.arcadedb.server.gremlin.GremlinServerPlugin " arcadedata/arcadedb:latest ``` as mentioned in docs, should i include the graph database also in Darcadedb.server.defaultDatabases ??

**Reply by lvca** [reply]
Could you please list the graph directory? It could be empty and it would explain why the studio doesn't enlist it.

**Reply by Praveenstein** [reply]
![Screenshot 2022-12-08 120831](https://user-images.githubusercontent.com/45847217/206376161-c8f1495b-98b7-42a5-9380-d48517be59df.png)
here is the screenshot

**Reply by lvca** [reply]
schema.json is there, so it should definitely be enlisted between the available databases. It's a bug. I've opened a new issue from this discussion: https://github.com/ArcadeData/arcadedb/issues/662. Working on it.
